app/config/config_test.yml

```diff
@@ -1,5 +1,4 @@
 parameters:
-    AdapterSecurityAdminClass:                LegacyTests\PrestaShopBundle\Mock\AdapterSecurityAdminMock
     prestashop.security.voter.product.class:  LegacyTests\PrestaShopBundle\Mock\PageVoter
```

Comment:
I just noticed 😄 why "product" in this class name ? Who thought this was relevant 😝

Reply:
:man_shrugging: